ocaml/xapi/xapi_network.ml

```diff
@@ -329,3 +329,42 @@ let with_networks_attached_for_vm ~__context ?host ~vm f =
         error "Caught %s while detaching networks" (string_of_exn e)
     end;
     raise e
+
+(*
```

Comment:
I can't get used to adding commented out code...

Reply:
Sorry, yes. (An artefact of the development process, left over in a pull-request made when tired...)